src/DynamoDb.Ok/Library.fs

```diff
@@ -165,6 +165,14 @@ module ExpressionAttributeName =
 
         attributeName, (attributeName, value) :: attributes
 
+    let getNameAlias names actualName =
```

Comment:
This supports up to 26 unique field names (using single letters) . Thanks to Junie